process/README.md

```diff
@@ -1,7 +1,7 @@
 
 # I. Overview
 
-This document describes the Open Source Security Foundation (OpenSSF) lifecycle process for Technical Initiatives (TI) which include Working Groups (WG), Projects (developing code or specifications), and Special Interest Groups (SIG).
+This document describes the Open Source Security Foundation (OpenSSF) lifecycle process for Technical Initiatives (TI) which include Working Groups (WG) (non-software focused), Projects (developing code or specifications), and Special Interest Groups (SIG). 
```

Comment:
First, given our change in the definition of projects, this should at least be:
```suggestion
This document describes the Open Source Security Foundation (OpenSSF) lifecycle process for Technical Initiatives (TI) which include Working Groups (WG) (non-software and non-specification focused), Projects (developing code or specifications), and Special Interest Groups (SIG). 
```
But I don't know that this is even correct because [the project lifecycle reads](https://github.com/ossf/tac/blob/main/process/project-lifecycle.md#sandbox-entry-requirements-and-considerations):

> It is expected that the initial code or specification developed by an OpenSSF WG be kept within their repository and will not function as a Project in its own right. Should the initial WG code or specification grow and mature that it warrants its own Project status, then it is subject to Sandbox entry requirements.

So, while the primary focus of a WG may not be to develop software or specs it can still do some of that so it doesn't seem to be a useful way to distinguish WGs.